public/app/features/query/state/runRequest.test.ts

```diff
@@ -109,6 +109,26 @@ function runRequestScenario(desc: string, fn: (ctx: ScenarioCtx) => void) {
   });
 }
 
+function runRequestScenarioThatThrows(desc: string, fn: (ctx: ScenarioCtx) => void) {
+  describe(desc, () => {
+    const ctx = new ScenarioCtx();
+    let consoleSpy: jest.SpyInstance<any>;
+
+    beforeEach(() => {
+      consoleSpy = jest.spyOn(console, 'error').mockImplementation(() => {});
+      setEchoSrv(new Echo());
+      ctx.reset();
+      return ctx.setupFn();
```

Comment:
Just curious why `return` is needed here?

Reply:
No idea, it was already like that.